pkg/pgmodel/migrate.go

```diff
@@ -111,18 +111,34 @@ func Migrate(db *pgxpool.Pool, versionInfo VersionInfo) (err error) {
 	return nil
 }
 
-//CheckDependcies makes sure all project dependencies are set up correctly
+// CheckDependencies makes sure all project dependencies, including the DB schema
+// the extension, are set up correctly. This will set the ExtensionIsInstalled
+// flag and thus should only be called once, at initialization.
 func CheckDependencies(db *pgxpool.Pool, versionInfo VersionInfo) (err error) {
+	conn, err := db.Acquire(context.Background())
+	if err != nil {
+		return fmt.Errorf("failed to check schema version: %w", err)
```

Comment:
it can't be; it has an interior error it needs to wrap (I don't think it's worth creating it's own error type, realistically we're never going to branch on it)